src/AppBundle/Entity/Base/PostalAddress.php

```diff
@@ -50,15 +51,27 @@
      * @Groups({"place"})
      * @ORM\Column(nullable=true)
      * @Assert\Type(type="string")
+     * @Assert\NotBlank(groups={"delivery_address"})
      * @ApiProperty(iri="https://schema.org/name")
      */
     private $name;
 
     /**
+     * @var string Family name. In the U.S., the last name of an Person. This can be used along with givenName instead of the name property.
+     *
+     * @ORM\Column(nullable=true)
+     * @Assert\Type(type="string")
+     * @Assert\NotBlank(groups={"delivery_address"})
+     * @ApiProperty(iri="https://schema.org/familyName")
+     */
+    private $familyName;
```

Comment:
Is the `familyName` really helpful? This is an address.
Some people may get delivered at "Acme Company" address.

Are we ok that `name` should the name of the item, i.e. "Home", "Work"?
I think everything can be managed through the `description` field for additional details